status-checker.js

```diff
@@ -18,6 +18,13 @@ import { checkIfNoMovesLeft } from './board-printer.js';
     Otherwise, return false
 */
 function checkRow(board, player, rowNumber) {
+  const checkAllThreeSquares = [];
```

Comment:
Can we use a boolean instead of an array? So we can simplify the code.

Reply:
Do you refer hardcoding+if statement?